mlflow/__init__.py

```diff
@@ -24,21 +24,22 @@
 
 For a lower level API, see the :py:mod:`mlflow.tracking` module.
 """
-
 from mlflow.version import VERSION as __version__
+from mlflow.utils.logging_utils import _configure_mlflow_loggers
+import mlflow.tracking.fluent
 
 # Filter annoying Cython warnings that serve no good purpose, and so before
 # importing other modules.
 # See: https://github.com/numpy/numpy/pull/432/commits/170ed4e33d6196d7
 import warnings
 warnings.filterwarnings("ignore", message="numpy.dtype size changed")  # noqa: E402
 warnings.filterwarnings("ignore", message="numpy.ufunc size changed")  # noqa: E402
+# log a deprecated warning only one per module
```

Comment:
Hyper nit: *once* per module

Reply:
Also, is this truly once per module? I tried adding the `deprecated` decorator to `mlflow.pyfunc.save_model` and then called *both* `mlflow.pyfunc.load_pyfunc` and `mlflow.pyfunc.save_model`. Both function calls yielded a deprecation warning.

Reply:
Thanks! It's once per module per function - as in, repetitions of the warning for a particular function occur only once per module. If there are multiple deprecated functions in a single module, they will each trigger a warning.